contracts/terraswap_token/src/contract.rs

```diff
@@ -28,7 +26,7 @@ pub fn instantiate(
     msg.validate()?;
 
     // create initial accounts
-    let total_supply = create_accounts(&mut deps, &msg.initial_balances)?;
+    let total_supply = create_accounts(&mut deps, msg.initial_balances.as_slice()).unwrap();
```

Comment:
is `unwrap` better than `?`macro?

Reply:
https://github.com/terraswap/terraswap/pull/44/commits/99bf6d48a6fe53f3962b7ba3787f9ab58842c759